node_modules/.package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
There shouldn't be UI related files in the root of the repo. Same with the next two files.